external/include/libcxx/csignal

```diff
@@ -18,7 +18,7 @@
 //***************************************************************************
 // include/libcxx/csignal
 //
-//   Copyright (C) 2009 Gregory Nutt. All rights reserved.
+//   Copyright (C) 2009, 2016 Gregory Nutt. All rights reserved.
```

Comment:
Why do you change the header?

Reply:
this change was from nuttx backport cl only, it was updated then